cfgov/login/urls.py

```diff
@@ -55,30 +51,11 @@
             url=reverse_lazy("wagtailadmin_logout"), query_string=True
         ),
     ),
-    # Override Wagtail and Django password views to enforce our password policy
-    re_path(
-        r"^django-admin/password_change",
-        change_password,
-        name="django_admin_account_change_password",
-    ),
-    path(
-        "admin/password_reset/confirm/<uidb64>/<token>/",
-        CFGOVPasswordResetConfirmView.as_view(),
-        name="wagtailadmin_password_reset_confirm",
-    ),
+    # Redirect Django's password change view to Wagtail's
```

Comment:
Something I didn't catch on previous reviews of this PR: in the current implementation, you can change your password from within the Django admin (you see a form and can enter your current password and new one). With this change, because we disable that view, you can only get redirected to the Wagtail password reset view, which asks you for your email address, sends the email, and then requires you to open the email to get the unique link which you can then use to change your password.

This is slightly less user-friendly; with SSO coming it seems fine to adopt this change (especially since it's only for the _Django_ admin change password view, the Wagtail user profile one still works without an email flow), but I wanted to make a note.